Source/WebKit/WebProcess/Network/NetworkProcessConnection.cpp

```diff
@@ -83,21 +83,21 @@ NetworkProcessConnection::NetworkProcessConnection(IPC::Connection::Identifier&&
     : m_connection(IPC::Connection::createClientConnection(WTFMove(connectionIdentifier)))
     , m_cookieAcceptPolicy(cookieAcceptPolicy)
 {
-    m_connection->open(*this);
+    protectedConnection()->open(*this);
 
     if (WebRTCProvider::webRTCAvailable())
-        WebProcess::singleton().libWebRTCNetwork().setConnection(m_connection.copyRef());
+        WebProcess::singleton().protectedLibWebRTCNetwork()->setConnection(m_connection.copyRef());
 }
 
 NetworkProcessConnection::~NetworkProcessConnection()
 {
-    m_connection->invalidate();
+    protectedConnection()->invalidate();
 }
 
 bool NetworkProcessConnection::dispatchMessage(IPC::Connection& connection, IPC::Decoder& decoder)
 {
     if (decoder.messageReceiverName() == Messages::WebResourceLoader::messageReceiverName()) {
-        if (auto* webResourceLoader = WebProcess::singleton().webLoaderStrategy().webResourceLoaderForIdentifier(AtomicObjectIdentifier<WebCore::ResourceLoader>(decoder.destinationID())))
+        if (RefPtr webResourceLoader = WebProcess::singleton().protectedWebLoaderStrategy()->protectedWebResourceLoaderForIdentifier(AtomicObjectIdentifier<WebCore::ResourceLoader>(decoder.destinationID())))
```

Comment:
We don't need `protectedWebResourceLoaderForIdentifier` here, `webResourceLoaderForIdentifier` is fine.